cmd/createProjects.go

```diff
@@ -192,6 +193,39 @@ func (p *Project) createProject(repo, projectName string, force bool, overwrite
 		qwe(ExitCodeError, err, "failed to parse the scope-included-files flag")
 	}
 
+	criticality, err := p.cmd.Flags().GetString("criticality")
+	if err != nil {
+		qwe(ExitCodeError, err, "failed to parse the criticality flag")
+	}
+
+	businessCriticality := func() string {
+		const (
+			criticalityMajor  = "Major"
+			criticalityHigh   = "High"
+			criticalityMedium = "Medium"
+			criticalityLow    = "Low"
+			criticalityNone   = "None"
+			criticalityAuto   = "Auto"
+		)
+
+		switch strings.ToLower(criticality) {
+		case strings.ToLower(criticalityMajor):
```

Comment:
Why don't you define business critically constants in lowercase?

Reply:
We're processing as Capitalized on Kondukto side. I needed decide to process here or Kondukto side and I choosed here.

Reply:
The criticality levels can be renamed on the Kondukto side, so the customer can rename the Low one as MyLow. 
We should implement the criticality fields based on level numbers like level1, level2 ...